packages/cli/src/index.ts

```diff
@@ -197,8 +299,7 @@ cli.on('command:*', async () => {
 // We only keep the original code environment.
 preventExperimentalWarning();
 
-cli.help();
-
-cli.version(version);
-
-cli.parse();
+// default to start a development server
+if (process.argv.slice(2).length === 0)
```

Comment:
There is a problem with this way of writing. If I follow arg, it will also be used as a parameter. If I use farm -m directly, it will not work.

Reply:
I think this way of writing is not good. Doesn’t citty provide this direct operation method?

Reply:
> I think this way of writing is not good. Doesn’t citty provide this direct operation method

`citty` doesn't provide a default behavior, it means if I run farm-cli with empty args (for example just `pnpm farm`), `citty` will throw an error to tell you that no args provided. But the current version of farm-cli will start a dev server by default.

I tried to add a default handler in `main`, but `citty` will run it twice with subcommand given. This means even you are running `pnpm farm clean`, a dev server will be started soon after cleaned.